reana_server/reana_admin/main.py

```diff
@@ -639,3 +640,61 @@ def set_default_quota_limit(ctx):
             resource_type=resource.type_.name,
             limit=DEFAULT_QUOTA_LIMITS.get(resource.type_.name),
         )
+
+
+@reana_admin.command("consume-queue")
+@click.option(
+    "--queue-name",
+    "-q",
+    required=True,
+    type=str,
+    help="Name of the queue that will be consumed, e.g workflow-submissions",
+)
+@click.option(
+    "--key",
+    "-k",
+    type=str,
+    help="Key that will be used to get the id value from the message body, e.g workflow_name_or_id",
+)
+@click.option(
+    "--messages-to-delete",
+    "-m",
+    multiple=True,
+    help="List of message ids to remove from the queue, e.g UUID of a workflow",
+)
+@click.option(
+    "-i",
+    "--interactive",
+    is_flag=True,
+    default=False,
+    help="Manually decide which messages to remove from the queue.",
+)
+def consume_queue(
```

Comment:
Some of the other `reana-admin` commands require the admin access token to be provided by the user (e.g. [`token-grant`](https://github.com/reanahub/reana-server/blob/a42b1efeb722c19181d6fe016c0e308cbafa134c/reana_server/reana_admin.py#L215)), but not all of them. Should we check it in every command? In this case, we can open an issue. /cc @tiborsimko

Reply:
I have no idea. The `consume-queue` command doesn't require `admin token` so I did not add it.

Reply:
Related issue: https://github.com/reanahub/reana-server/issues/489